src/lib/y2partitioner/widgets/format_and_mount.rb

```diff
@@ -440,15 +441,7 @@ def label
       end
 
       def items
-        Y2Storage::Filesystems::Type.all.select { |fs| supported?(fs) }.map do |fs|
-          [fs.to_sym, fs.to_human_string]
-        end
-      end
-
-    private
-
-      def supported?(fs)
-        SUPPORTED_FILESYSTEMS.include?(fs.to_sym)
```

Comment:
SUPPORTED_FILESYSTEM still exists in this file.

Reply:
I forgot to remove it. Thanks.